app/src/main/res/values/strings.xml

```diff
@@ -59,22 +59,22 @@
     <string name="importOptionFilesystemTitle">Import from filesystem</string>
     <string name="importOptionFilesystemExplanation">Choose a specific file from the filesystem.</string>
     <string name="importOptionFilesystemButton">From filesystem</string>
-    <string name="importOptionApplicationTitle">Use external app</string>
+    <string name="importOptionApplicationTitle">Use another app</string>
     <string name="importOptionApplicationExplanation">Use any app or your favorite file manager to open a file.</string>
-    <string name="importOptionApplicationButton">Use external app</string>
+    <string name="importOptionApplicationButton">Use another app</string>
 
     <string name="about">About</string>
-    <string name="app_copyright_fmt" translatable="false" tools:ignore="PluralsCandidate">Copyright 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
-    <string name="app_copyright_old">Based on Loyalty Card Keychain, copyright 2016–2020 Branden Archer.</string>
+    <string name="app_copyright_fmt" tools:ignore="PluralsCandidate">Copyright © 2019–<xliff:g>%d</xliff:g> Sylvia van Os.</string>
+    <string name="app_copyright_old">Based on Loyalty Card Keychain\ncopyright © 2016–2020 Branden Archer.</string>
     <string name="app_license">Copylefted libre software, licensed GPLv3+.</string>
     <string name="about_title_fmt">About <xliff:g id="app_name">%s</xliff:g></string>
     <string name="debug_version_fmt">Version: <xliff:g id="version">%s</xliff:g></string>
     <string name="app_revision_fmt">Revision Info: <xliff:g id="app_revision_url">%s</xliff:g></string>
-    <string name="app_libraries"><xliff:g id="app_name">%s</xliff:g> uses the following third-party libraries: <xliff:g id="app_libraries_list">%s</xliff:g></string>
-    <string name="app_resources"><xliff:g id="app_name">%s</xliff:g> uses the following third-party resources: <xliff:g id="app_resources_list">%s</xliff:g></string>
+    <string name="app_libraries">Libre software libraries: <xliff:g id="app_libraries_list">%s</xliff:g></string>
+    <string name="app_resources">Libre resources: <xliff:g id="app_resources_list">%s</xliff:g></string>
```

Comment:
I still want both of these to just be "Third-party" instead of "Libre" because it's about showing respect to authors of code we use.